### DISCUSSION TITLE
Happy Birthday BirdNET-Pi!!!

### DISCUSSION THREAD

**Discussion by nilspupils** [mcguirepr89/BirdNET-Pi#578] [General]
So this: 

https://github.com/mcguirepr89/BirdNET-Pi/discussions/11#discussion-3595354

was one year ago today! I think it is time to celebrate this great project and to thank @mcguirepr89 for his awesome work!
Thank you Patrick and all collaborators!!!

Nils

**Comment by mcguirepr89** [comment]
Thanks, Nils -- this is a very kind gesture. I'm emailing you now with a little info. 

My best regards as always,
Patrick

**Reply by jflyons** [reply]
Great work! I have only discovered in recent weeks but am very grateful to have found the solution and community!

**Reply by tomwhite101** [reply]
Happy Birthday Birdnet-Pi and thanks @mcguirepr89 for this super project!

Tom

**Reply by Fish31520** [reply]
Thanks @mcguirepr89 for your incredible work !

**Reply by blogdemoi** [reply]
Merci ! from France for this wonderful project.

**Reply by DCS17** [reply]
Thanks @mcguirepr89 and contributers. Happy Birthday (a couple fo days late !)

**Reply by croisez** [reply]
I would say: Happy Bird-day ;-)
